src/index.css

```
@@ -4,63 +4,17 @@
-
-  /* NEUTRAL */
-  --color-background: 8, 10, 16;
-  /* #080A10 - Softer black */
-  --color-surface: 18, 22, 30;
-  /* #12161E - Softer surface */
-  --color-text: 255, 255, 255;
-  /* #FFFFFF */
-  --color-text-dark: 20, 20, 30;
-  /* #14141E */
-}
+@import "./styles/themes.css";
```

Stylelint (17.13.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)